src/components/Navigation/NavigationItem/NavigationItem.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { px } from '../../../utils';
+import classNames from 'classnames';
+import Link from '../../Link/Link';
+import { LinkVariants } from '../../Link/utils';
+import { HeaderContext } from '../../Header/Header';
+
+export interface NavigationItemProps extends React.HTMLAttributes<HTMLElement> {
```

Comment:
Once you spread the props onto the `<li` you can make this more specific with


```suggestion
export interface NavigationItemProps extends React.HTMLAttributes<HTMLLIElement> {
```